thirdweb/contracts/custom.py

```diff
@@ -100,21 +102,56 @@ def __init__(
         self.nft = self._detect_erc_721()
         self.edition = self._detect_erc_1155()
 
-        # self.signature_mint = self.detect_signature_mint()
+    def call(self, fn: str, *args) -> Any:
```

Comment:
Nice. Add a doc + example here too ?

Reply:
This class is internal for now on python - that's why the docs are manual in the other place, reason being that I prefer to have a handwritten doc for something as important as custom contract + people don't really need to see the generic custom contract type externally.

Reply:
We could def expose it though if necessary

Reply:
but the `call` function itself needs some doc no? why would this class be internal?